app.py

```diff
@@ -0,0 +1,140 @@
+from flask import Flask, render_template, session,json
+from flask import request
+from flask import jsonify
+from heperfunctions import *
+from constants import *
+from flask import Response
+from psycopg2 import IntegrityError
+import logging
+
+
+# logging.basicConfig(filename='LogFolder/app_los.log' ,level = logging.DEBUG)
+
+app = Flask(__name__)
+logger = get_logger(__name__)
+conn = create_db_connection()
+
+
+@app.route('/getAllClients')
```

Comment:
Good Work, use extended link like for client's api 
use /client/v1/get_all_clients
don't use CAMEL format, use lower space with '_' as a Space

Reply:
This a Test check Please Ignore